pkg/commands/options/publish.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Google LLC All Rights Reserved.
+// Copyright 2021 Google LLC All Rights Reserved.
```

Comment:
I'm not sure why the boilerplate check is complaining about this, but just to appease it can you use the `/* */` comment form? 🙄

Reply:
Done. It doesn't match the other copyright headers, but 🤷